facts/config.py

```diff
@@ -32,13 +35,34 @@
 """
 Default configuration.
 
-The configuration `cfg_defaults` can be superseded with that read from `cfg_filename`
-(at path `../facts_conf/<progname>.cfg`), if file exists.
+The configuration `cfg_defaults` are loaded from `cfg_filename`, if file exists.
+** CentOS ** --> /etc/sysconfig/fiware-facts.cfg
+** Ubuntu ** --> /etc/default/fiware-facts.cfg
+** OS X ** --> /etc/defaults/fiware-facts.cfg
+
```

Comment:
Not a good idea using such "system" locations, considering fiware-facts is a PyPI package and not a Linux service. Maybe something like ``/etc/fiware.d/fiware-facts.cfg`` would be preferable (and this would allow us adding more FIWARE-related configuration files and even a "global" ``/etc/fiware.conf`` configuration file).